Sources/SharedModels/Verification.swift

```diff
@@ -61,12 +69,16 @@ public func verify(
       // TODO: this score should be computed from the string rather than using what is handed us.
       //       in fact maybe we need an ArchivableMove to remove that info?
       return .init(cubeFaces: cubeFaces, foundWord: foundWord, score: move.score)
-    } else {
-      return nil
-    }
+    } else { return nil }
 
   case let .removedCube(point):
-    if puzzle[point].isInPlay {
+    if
+      puzzle[point].isInPlay
+        // NB: Allow "removing" an out of play cube if it was removed in the previous move. This
+        //     is to work around a race condition in the client where quickly tapping multiple times
+        //     can accidentally remove a single cube twice.
+        || (moveIndex > 0 && moves[moveIndex - 1].type == move.type)
+    {
```

Comment:
this will make the server accept games that are submitted with consecutive removals of the same cube.